### PR TITLE
RepoList 

### PR DESCRIPTION
**Pull request by jackpearsall**

### CODE REVIEW

File: src/components/repo-card.jsx [hunk 1]
```diff
@@ -1,5 +1,15 @@
 import React from 'react';
 
-const RepoCard = (props) => <li>Repo Card for: {props.username}</li>;
+const RepoCard = (props) => {
+  return (
+    <div>
+      <li>Repo Name: {props.name}</li>
+      <li>Repo Description: {!props.description ? (<span>There is no description</span>) : props.description}</li>
```
Comment (ersel) [inline src/components/repo-card.jsx]: nice use of ternary render

File: src/components/repo-list.jsx [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
+class RepoList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {},
+    };
+  }
+
+  componentDidMount() {
+    const DATA_URL = `https://mcr-codes-cohorts.herokuapp.com/users/${name}`;
```
Comment (ersel) [inline src/components/repo-list.jsx]: name could be read from `userName` prop.

File: src/components/repo-list.jsx [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
+      });
+  }
+
+  render() {
+    const { repos } = this.state.data;
+    return (
+      <div>{ !repos ? <div>loading</div> : repos.map((repo, index) => {
+        return (
+          <RepoCard
+            key={index}
```
Comment (ersel) [inline src/components/repo-list.jsx]: using index as key is discouraged, is there nothin in this list that can be used as a unique identifier?

Comment (tbilbe) [reply src/components/repo-list.jsx]:
```suggestion
            key={repo.name} 
```
We use unique repo names for all our repos

**Pull request merged**